### DISCUSSION TITLE
Aspiratv installation

### DISCUSSION THREAD

**Discussion by player041** [community/community#70267] [Programming Help]
### Body

The aspiratv apps was installed on my NAS server but nothing happen when I try to open it. I think something missing. Thx

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Dineth-H** [comment]
I'm not familiar with the aspiratv app, but I can give you some general troubleshooting tips for NAS apps:

- [ ] Make sure the app is installed correctly.
- [ ] Make sure the app is running.
- [ ] Make sure you have permission to use the app.
- [ ] Make sure your NAS server is properly configured for network access.
- [ ] **Make sure the app is compatible with your NAS server and other devices.**

If you've checked all of these things and the app is still not working, I recommend contacting the app's developer for further assistance.

Let me know if you have any other questions.
